third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1967,6 +1967,10 @@ func checkMatch(attributes map[string]string, attr string, gcp interface{}) stri
 }
 
 func checkListMatch(attributes map[string]string, attr string, gcpList []string) string {
+	if attributes[attr+".#"] == "" {
```

Comment:
What's going on here, exactly? Should we add a comment?

Reply:
A bunch of the `TestAccDataprocJob_*` tests fail without this. I followed up with an sdkv2 team member, wondering what changed and he said it's likely an inaccuracy that happens when shimming the `terraform-json` representation of state back to the old framework's representation of state. So, in the past we would get `x.# = 0` whereas now we get `x.# = ''`. It was likely not intentional, however, shouldn't be a big problem - but if we notice it is they can address it.

Reply:
Can you comment that inline?